### DISCUSSION TITLE
Yoast SEO absolutely incompatible with qtranslate xt

### DISCUSSION THREAD

**Discussion by andreysneg** [qtranslate/qtranslate-xt#1257] [General]
Yoast absolutely incompatible with qtranslate xt.

Snippets, sitemap... nothing dont work.

You wrote that you did something or I'm wrong ...

**Comment by herrvigg** [comment]
> You wrote that you did something or I'm wrong ...

Not sure what you are referring to.
I don't have time to work on this, some developer should take care of it. There are many other tasks that are more urgent from the project perspective.
I will mark this module as "experimental" or "degraded" until then.

**Reply by herrvigg** [reply]
Marked degraded. Please don't use discussion to report issues. Create new issues to track those that are not reported yet.
